lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -0,0 +1,233 @@
+
+        // Issue
+        configureTextFieldBorder(textField: issueTextField)
+        issueTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Issue Number")
+
+        // Publisher
+        configureTextFieldBorder(textField: publisherTextField)
+        publisherTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Marvel, DC Comics, Image, Dark Horse…")
+
+        // Release Date
+        configurePickerUIButton(button: selectADateButton)
+
+        // Genre
+        configurePickerUIButton(button: selectAGenreButton)
+
+        // Condition
+        configurePickerUIButton(button: selectAConditionButton)
+    }
+
+    private func configureTextFieldBorder(textField: UITextField) {
+        let bottomBorder = CALayer()
+        bottomBorder.frame = CGRect(x: 0.0, y: textField.frame.height - 3.0, width: textField.frame.width, height: 1.0)
+        bottomBorder.backgroundColor = LucienTheme.silver.cgColor
+        textField.borderStyle = UITextBorderStyle.none
+        textField.layer.addSublayer(bottomBorder)
+    }
+
+    private func configurePickerUIButton(button: UIButton) {
+        button.setImage(UIImage(named: "dropDownArrow"), for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets.left = button.frame.width - 24
+        button.tintColor = LucienTheme.dark
+    }
+
+    private func configureAddCoverUIButton(button: UIButton) {
+        button.titleLabel?.lineBreakMode = .byWordWrapping
+        button.titleLabel?.textAlignment = .center
+        button.setTitle("Take \n Cover \n Photo", for: .normal)
+        button.titleLabel?.font = LucienTheme.Fonts.Muli.bold.getFont(size: 13)
+        button.setTitleColor(LucienTheme.dark, for: .normal)
+        button.titleEdgeInsets.top = 25
+    }
+
+    private func createPlaceHolderAttributedString(textFieldPlaceholderText: String) -> NSAttributedString {
+        let placeholderAttributes = [
+        NSAttributedStringKey.foregroundColor: LucienTheme.silver,
+        NSAttributedStringKey.font : LucienTheme.Fonts.Muli.regular.getFont(size: 16) ?? UIFont()
+        ] as [NSAttributedStringKey : Any]
+       return NSAttributedString(string: textFieldPlaceholderText, attributes: placeholderAttributes)
+    }
+
+    @objc private func comicTitleEditingChanged(_ textField: UITextField) {
+        guard let comicTitle = textField.text, !comicTitle.isEmpty else {
+            finishButton.isEnabled = false
+            finishButton.tintColor = LucienTheme.finishButtonGrey
+            return
+        }
+        finishButton.isEnabled = true
+        finishButton.tintColor = LucienTheme.dark
+    }
+
+    // MARK: - IBOutlet Methods
+
+    @IBAction func selectADateButtonTapped(_ sender: UIButton) {
+        UIView.animate(withDuration: 0.3, animations: {
```

Comment:
This should be formatted differently, when you have multiple closures, each argument should be on its own line indented once from the top line e.g.
```
UIView.animate(
    withDuration: 0.3, 
    animations: {
        self.datePicker.isHidden = false
    },
    completion: { _ in
        let offset = CGPoint(x: 0, y: self.scrollView.contentSize.height - self.scrollView.bounds.size.height)
        self.scrollView.setContentOffset(offset, animated: true)
    }
)
```